README.md

```diff
@@ -12,25 +12,27 @@ Data assimilation Integration
 (JEDI)](https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/)
 framework.
 
+Documentation is available at https://hpc-stack.readthedocs.io/en/latest/. 
+
 This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS) project.
 
 ## Authors
 
 Rahul Mahajan, Kyle Gerheiser, Dusan Jovic, Hang-Lei, Dom Heinzeller
 
-Code Manager: Kyle Gerheiser
+Code Manager: Dom Heinzeller
```

Comment:
If this is for hpc-stack, it should not be me ... @edwardhartnett who should be that person?

Reply:
@climbfuji I updated CM to Alex Richert per @edwardhartnett 's email response to me:

> Kyle's replacement is Alex Richert and you can add him as the code maintainer.